src/Tizen.Multimedia.Remoting/MediaController/MediaControlMetadata.cs

```diff
@@ -116,5 +125,97 @@ internal MediaControlMetadata(IntPtr handle)
         /// </summary>
         /// <since_tizen> 4 </since_tizen>
         public string AlbumArtPath { get; set; }
+
+        /// <summary>
+        /// Gets or sets the season information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public (int number, string title) Season
```

Comment:
I am aware that tuples are a thing since C# 7.0, but I suggest using a single return value for better API readability. If you want to return two values, please create a structure, which aggregates both of them.

Reply:
Tizen support C# 7.x and we can use those features. and I don't think tuple make API readability worse. I'll add more description about it.

Reply:
I'll consider struct as return value type for extendibility, because it may add more parameter in the future. With tuple, we can't add more return parameter without deprecation of current API.